### PR TITLE
www.rere.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/105352

<details><summary>Screenshot:</summary>

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ matome.eternalcollegest.com,ushigyu.net,nilcollection.net,gamers-info.net,appl-y
 ! `###fix_sidebar`
 sugi9532.com,frozen.analyst.jp,5app-ch.com,genshinnews.apexlegends-leaksnews.com###fix_sidebar
 ! `##.code-block`
-bazurecipe.com,frozen.analyst.jp,entano.jp,mainoriti.com,genshinnews.apexlegends-leaksnews.com,crossbike.biz,comicspoil007.com,yoso-walk.net,poire122.com,automaton-media.com,pontako.com##.code-block
+rere.jp,bazurecipe.com,frozen.analyst.jp,entano.jp,mainoriti.com,genshinnews.apexlegends-leaksnews.com,crossbike.biz,comicspoil007.com,yoso-walk.net,poire122.com,automaton-media.com,pontako.com##.code-block
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:22]: This blocks thier popup bar on mobile (not sure if is annoyances @zloyden ). I'd go with
```
rere.jp##.rectangle_bnr
rere.jp##.singleTopBnr
```
Note there's already `kashi-mo.com##.rectangle_bnr`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:22]: > This blocks thier popup bar on mobile

@Yuki2718 unfortunately, I don't have it. Can you show what it block?

>
```
rere.jp##.rectangle_bnr
rere.jp##.singleTopBnr
```

I agree with these rules if they are needed.

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:22]: This one but is already blocked by `##div[data-insertion]` in Base.

<details>
<summary>Screenshot</summary>

rere

</details>
It links to
```
https://www.rere.jp/beginners/106752/
https://www.rere.jp/beginners/106749/
https://www.rere.jp/beginners/106921/
```
but having looked carefully, they seem to be native-ad articles. So I backtrack my words, `##.code-block` will be fine.

Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:22]: Ok, I got it. Thanks for the explanation :)

**Review by zloyden — APPROVED** [review]

**Review by Yuki2718 — APPROVED** [review]

**Pull request merged**